crates/sdk/Cargo.toml

```diff
@@ -145,6 +145,7 @@ tokio = { workspace = true, features = ["full"] }
 
 [target.'cfg(target_family = "wasm")'.dependencies]
 tokio = { workspace = true, default-features = false, features = ["sync"] }
+getrandom = { version = "0.3", features = ["wasm_js"] }
```

Comment:
my only nit is adding `getrandom` to the workspace's `Cargo.toml`, then here using `workspace = true`